packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/entity-editor/properties-section/property-table/use-row-data/generate-property-rows-from-entity/generate-property-row-recursively.ts

```diff
@@ -35,14 +35,21 @@ import { getExpectedTypesOfPropertyType } from "./get-expected-types-of-property
  *
  * @returns property row (and nested rows as `children` if it's a nested property)
  */
-export const generatePropertyRowRecursively = (
-  properties: unknown,
-  propertyTypeBaseUri: string,
-  propertyKeyChain: string[],
-  entitySubgraph: Subgraph<SubgraphRootTypes["entity"]>,
-  requiredPropertyTypes: string[],
+export const generatePropertyRowRecursively = ({
+  propertyTypeBaseUri,
+  propertyKeyChain,
+  entity,
+  entitySubgraph,
+  requiredPropertyTypes,
   depth = 0,
-): PropertyRow => {
+}: {
+  propertyTypeBaseUri: string;
+  propertyKeyChain: string[];
+  entity: Entity;
+  entitySubgraph: Subgraph<SubgraphRootTypes["entity"]>;
+  requiredPropertyTypes: string[];
```

Comment:
Similar comment actually for all of these, can we try and use more descriptive types, I'm assuming one of `BaseUri` or `VersionedUri`

Reply:
Thanks, updated here https://github.com/hashintel/hash/pull/1593/commits/00f0ca00e7e6aed5be438497ddabbef730b0e97d